### PR TITLE
Resource history docs

### PR DESCRIPTION
**Pull request by nstokoe**
This is a first pass on the documentation for resource history. 

closes HB-8084

### CODE REVIEW

File: pages/docs/project-management/resource-history.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+description: View previous versions of your project resources with immutable resources.
+---
+
+# Resource history
+
+Your project's resource history allows you to review how the configuration of your resources resources has changed, alongside who made the change and when.
```
Comment (luchodev) [inline pages/docs/project-management/resource-history.mdx:7]: `your resources resources`

File: pages/docs/project-management/resource-history.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+To view the visual diffs for a grouped change, click on the specific row in the table and a modal will appear with the diffs for each resource.
+
+#### Who can access project-level resource history?
+If a user has permission to build resources from code (the `CREATE_DATA_MODEL_REPO_BUILD` permission), they are allowed to view the project-level resource history.
+
+
+## "Additional changes detected" warning
+
+When changes are shown in comparison to the previous version of the resource, sometimes there are changes that are shown in the comparison.
```
Comment (luchodev) [inline pages/docs/project-management/resource-history.mdx:50]: sometimes there are changes that are `not`(?) shown in the comparison.

File: pages/docs/project-management/resource-history.mdx (hunk 1 above)
Comment (nstokoe) [reply pages/docs/project-management/resource-history.mdx]: Thank you! good catch

File: pages/docs/project-management/resource-history.mdx (hunk 2 above)
Comment (nstokoe) [reply pages/docs/project-management/resource-history.mdx]: Yes, this is a typo as well. Thank you

File: pages/docs/project-management/resource-history.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- [Metrics](/docs/metrics)
+- [Saved explorations](/docs/visualizing-data/explorations#creating-a-saved-exploration)
+- [Dashboards](/docs/dashboards)
+- [Color Palettes](/docs/data-ops/config-schema/Color-Palette)
+- [Homepage Launchpad](/docs/data-ops/config-schema/Homepage-Launchpad)
+
+## Specific resource version history
+
+When viewing data models, saved explorations, dashboards and metrics, you can quickly view the history of that resource over time.
+- For a specific **data models**, **saved explorations**, or **dashboards** you can view the history of that resource by navigating to the appropriate page from the
```
Comment (annanixon) [inline pages/docs/project-management/resource-history.mdx:20]: nit: For ~~a~~ specific data models, saved explorations, or dashboards

**Review by annanixon — APPROVED** [review]
Looks great!

**Pull request merged**